curiouslearning/FeedTheMonsterJS
# fix: In the pause pop-up the buttons are loading before the background image of the pop-up. (FM-351)

# Changes
- popup image background load slow in low and mid devices

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Tab pause button
- Make sure the pause bg image in the pause popup loads seamlessly

Ref: [FM-351](https://curiouslearning.atlassian.net/browse/FM-351)


[FM-351]: https://curiouslearning.atlassian.net/browse/FM-351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
